src/assets/index.ts

```diff
@@ -0,0 +1,9 @@
+import IcLogoSim from "./svg/ic_logo_sim.svg?react";
+import IcLoud from "./svg/ic_loud.svg?react";
+import IcSym from "./svg/ic_sym.svg?react";
+import IcTopBar from "./svg/ic_top_bar.svg?react";
+
+export { IcLogoSim };
+export { IcLoud };
+export { IcSym };
+export { IcTopBar };
```

Comment:
이 부분은 
```suggestion
export { IcLogoSim, IcLoud,  IcSym, IcTopBar  };
```

요 방식이 더 좋겠네요!!!

Reply:
> 이 부분은
> 
> 요 방식이 더 좋겠네요!!!

수정 하겠습니다!